<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>atacksPage</class>
 <widget class="QDialog" name="atacksPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">border: none;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #717171, stop: 0.4 #626262,
stop: 0.9 #4c4c4c, stop: 1 #333333);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>590</width>
        <height>269</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutFrame"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTroopsGeneral">
     <property name="styleSheet">
      <string notr="true">QLabel {
  background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
}


QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FE1E1E, stop: 1 #FFFFFF);
    font-size:9px;
 }



 QSpinBox {
     padding-right: 15px; /* make room for the arrows */
     
	background-color: rgb(255, 255, 255);
     border-width: 3;
 }</string>
     </property>
     <property name="title">
      <string>Setare trupe general</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutTroopSettings_2">
      <item row="2" column="6">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_knight.png</pixmap>
        </property>
        <property name="buddy">
         <cstring>spinBoxKnight</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="4" rowspan="2" colspan="3">
       <widget class="QGroupBox" name="groupBoxRamTarget_2">
        <property name="title">
         <string>Tinta catapulta</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="comboBoxBuilding_2">
           <property name="currentIndex">
            <number>7</number>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAfterCurrent</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <item>
            <property name="text">
             <string>Cladirea Principala</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cazarma</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Grajd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Atelier</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Biserica</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Curte Nobila</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fierarie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Piata Centrala</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Statuie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Targ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Taietori de lemne</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mina de argila</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mina de fier</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ferma</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magazie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Zid</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="7" colspan="2">
       <widget class="QSpinBox" name="spinBoxKnight">
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="7" colspan="2">
       <widget class="QSpinBox" name="spinBoxSnob">
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxSword">
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_archer.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_spear.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_spy.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_ram.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_nobil.png</pixmap>
        </property>
        <property name="buddy">
         <cstring>spinBoxSnob</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_sword.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_light.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_catapult.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_axe.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_marcher.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/unit_heavy.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxSpear">
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxAxe">
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBoxArcher">
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBoxSpy">
        <property name="maximum">
         <number>50000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBoxLight">
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="spinBoxMarcher">
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="spinBoxHeavy">
        <property name="maximum">
         <number>30000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="spinBoxRam">
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="spinBoxCatapult">
        <property name="maximum">
         <number>30000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="7" colspan="2">
       <widget class="QPushButton" name="pushButtonOk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
     min-height:25px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }
 QPushButton:unchecked {
border: 2px solid #8f8f91;
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }
QPushButton:checked {
 border: 2px #ffffff;
border-radius: 6px;
background-color: color: rgb(0, 0, 0);
font-color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="7" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Trupe costum:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
